src/assets/src/components/common.tsx

```diff
@@ -39,6 +38,7 @@ interface RemoveButtonProps {
     disabled: boolean;
     size?: "block" | "lg" | "sm";
     screenReaderLabel: string;
+    children?: React.ReactNode;
```

Comment:
Just wondering `children` variable added now but it is referenced in code before, if I remove it gives a red wiggly line. Do you know why it worked before?

```
export const RemoveButton: React.FC<RemoveButtonProps> = (props) => {
    const className = "btn btn-danger " + (props.size ? ` btn-${props.size}` : "");
    const disabledMessage = props.disabled && DisabledMessage;
    return (
        <button onClick={() => props.onRemove()} disabled={props.disabled} className={className} aria-label={props.screenReaderLabel}>
            <FontAwesomeIcon icon={faTrashAlt} />
            {props.children}
            {disabledMessage}
        </button>
    );
}
```

Reply:
Sorry, I missed/forgot about this comment. I think this explains it: https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-typescript-definitions

Reply:
Thanks for Link. It now make sense why it was not an error before